source/org/zfin/uniprot/secondary/SecondaryTermLoadAction.java

```diff
@@ -0,0 +1,153 @@
+package org.zfin.uniprot.secondary;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.*;
+import org.apache.commons.lang3.ObjectUtils;
+import org.springframework.security.authentication.encoding.Md5PasswordEncoder;
+import org.zfin.sequence.ForeignDB;
+import org.zfin.uniprot.UniProtLoadLink;
+
+import java.util.*;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class SecondaryTermLoadAction implements Comparable<SecondaryTermLoadAction> {
+    public enum Type {LOAD, INFO, WARNING, ERROR, DELETE, IGNORE, DUPES}
+
+    public enum SubType {
+        MARKER_GO_TERM_EVIDENCE("MarkerGoTermEvidence", 1),
+        EXTERNAL_NOTE("ExternalNote", 2),
```

Comment:
use class objects for ease for handling later
 EXTERNAL_NOTE(ExternalNote.class, 2),
this validates the first entry at compile time. Otherwise, it may only fail at runtime if the string is wrong.